Dockerfile.template

```diff
@@ -102,6 +113,7 @@ RUN set -ex; \
 		socat \
 	; \
 	rm -rf /var/lib/apt/lists/*; \
+	[ -n "${REPOSITORY}" ] && rm /etc/apt/sources.list.d/mariadb.list; \
```

Comment:
Is this being removed because it won't work outside your environemtn (and thus will make `apt-get update` fail), or for privacy/secrecy? (build args get recorded in the image history, so if it's the latter this won't do much to protect it :grimacing: :smile:)